pycrostates/preprocessing/spatial_filter.py

```diff
@@ -0,0 +1,165 @@
+from typing import Union
+
+import mne
+import numpy as np
+from mne import BaseEpochs
+from mne.channels.interpolation import _make_interpolation_matrix
+from mne.io import BaseRaw
+from mne.io.pick import _picks_by_type
+from mne.parallel import parallel_func
+from mne.utils.check import _check_preload
+
+from ..utils._checks import _check_n_jobs, _check_type, _check_value
+from ..utils._docs import fill_doc
+from ..utils._logs import logger, verbose
+
+
+@fill_doc
+@verbose
```

Comment:
IIRC, you copied the `verbose` decorator from MNE which also fills the doc, so only one is needed.

```suggestion
@verbose
```